distribution/test/itests/test-itests-ddf/pom.xml

```diff
@@ -128,7 +128,7 @@
         <dependency>
             <groupId>ddf.distribution</groupId>
             <artifactId>sdk-app</artifactId>
-            <version>${project.version}</version>
+            <version>2.18.0</version>
```

Comment:
are we planning on making this an `ddf.sdk.version` at the root level?

Reply:
I can do that for this PR. Ideally we wouldn't need the property at all and not included in itests or the distribution since it really isn't production code. But that is more of a long term thought. Open to ideas on it.